software.md

```diff
@@ -0,0 +1,31 @@
+# FOSS United Community Software Policy
+This policy is devised to guide, but not mandate, software choices within the 
+FOSS United Community.
+
+- Define what we (the FOSS United Community) wish to do. Ask ourselves 
+whether what we wish to do will help in full-fulling the community's goals.
+- Define your requirements for the software solution clearly. Clearly 
+differentiate between needs and wants. Needs are features that are crucial to 
+performing the task at hand and wants are niceties, things we can do without.
+- Look for FOSS solutions that are capable of meeting the needs at a level 
+that's acceptable to the community. Focus should be on the needs.
+- Look at the cost in terms of pricing, effort and time that the community needs 
+to spend to be able to use solution and constantly look for ways we can lower 
+such barriers and consider supporting the project if that helps.
+- When a solution is available in the FOSS ecosystem use it.
+- When such a solution is not available in the ecosystem assuming that this is 
+the only thing that prevents us from doing something that will benefit the cause 
+of FOSS United then look for proprietary solutions.
+- Assess the costs of each solution in terms of privacy, freedom, pricing, effort 
+and time that the community needs to spend to be able to use software and then 
+make a choice as a community.
```

Comment:
I think the evaluation and decision making process should be separate. Looks like we are mixing of them here. I would prefer this to be done something like this:

1. Requirements are identified and documented
2. Possible choices are identified
3. A person or a team evaluates the possible options and documents their findings
4. there will be a discussion with the members of the community to make the final decision

Not sure if this looks too bureaucratic. I'm not even sure if it is feasible to evaluate all the possible options.

Reply:
Seperating them into sections does indeed make sense. I'll work on this.

Reply:
Regarding the feasibility of exploring all options, I think it's safe to assume that we won't be able to asses every possible solution under the sun. We can try and get as much information as we possibly can based on our circumstances. Regardless later on if we find something better than what we have now, we can always make the switch. I also hope that as time goes by we will get more efficient in making such decisions.